src/pages/hegel/reference/mechanical-object-section-1.mdx

```diff
@@ -0,0 +1,18 @@
+# The Mechanical Object – Section 1
+
+
+The first section opens with a paragraph that is a reflection on the “mechanical object” rather than a continuation of the immanent development of what the “mechanical object” is, as was the case in the previous paragraph. This is something that Hegel does often, and without any warning, and it is important to distinguish passages that add to the immanent development of a concept from passages that are reflections on that immanent development.  
+	
+With that caveat made, we can now look at what Hegel writes in the first paragraph of Section 1. The paragraph is focused on making one point about the nature of the “mechanical object”: that one should not try to conceive of the “mechanical object” as being made up of two different things. The examples that Hegel gives are, form and matter, whole and parts, and, substance and accidents. In other words, one could try to distinguish the form of a “mechanical object” from its matter. One could say, what it is for that rock to be a mechanical object is for the form of rock-ness to be in unity with the matter of a rock. This unity of form and matter is what constitutes the rock as a mechanical object. Hegel, however, warns us against any such conceptions. When critiquing the form-matter distinction, he writes, “such an abstract difference of individuality and universality is excluded by the Notion of object”  (712). The difference between form (universality) and matter (individuality) is a kind of difference that is “excluded” by the very concept of a mechanical object. Why is that? To answer this question we need only to cast our minds back to the first paragraph of “Mechanism”, where the “mechanical object” is explained to be an “immediate identity” such that it is a “universality that pervades the particularity and in it is immediate individuality” (711). In other words, it is incongruent with the concept of the “mechanical object” to conceive of it as having the difference of anything as the foundation for its being. What it is for it to be a “mechanical object” to be is for it to be an immediate identity. What is an immediate identity? The determinations of the “Concept” that constitute the “mechanical object”. One cannot conceptually distinguish the universal moment from the individual moment of the “mechanical object” because they are taken as identical to each other. The first sentence of the second paragraph of Section 1 attests to this clearly: “The object is therefore in the first instance *indeterminate*, in so far as it has no determinate opposition in it; for it is the mediation that has collapsed into immediate identity”  (712). The fact that the determinations of the “Concept” are immediately identical to each other and not mediated means that the “mechanical object” is indeterminate. In other words, it lacks determinacy because there is no difference within it for any determinacy to be established. 
```

Comment:
1. "Section 1." => What is Section 1? Are we in "A. Mechanical Object. I." ?
1. “mechanical object” => Why is this in quotation marks?
1. “such an abstract difference of individuality and universality is excluded by the Notion of object”  (712). => What citation system are you using here? This is something we should agree on sooner than later...
1. "To answer this question we need only to cast our minds back to the first paragraph of “Mechanism”," => was this anywhere discussed?
1. "immediate identity? The determinations of the “Concept” that constitute the “mechanical object”." => is this right? `immediate identity` sounds more like essence-talk rather than concept. I see that you go into explaining how it relates to the concept but it may be better to be up front that this relates to the concept rather than essence. 

Other than that, great paragraph! I particularly like the example with the rock, that's solid ;)

Reply:
1. I put "Section 1" because this section of Mechanism is begun with a number "1" at the beginning of the paragraph. After that there's a "Section 2" and a "Section 3". 
2. That's just something I carried over from my personal approach. It's a way of distinguishing when I'm writing about the determination within the logical development and when I'm writing about the determination in more general terms. I understand that that's a pretty big stylistic feature to implement. So if you think it's too much, I'm happy to drop it. 
3. I was trying to keep it simple. We all know that it's the Science of Logic - though it should probably state somewhere that it's Miller's translation- and so I just put the page number. That said, I do think it's probably a good idea to always footnote the original German when citing so as to have some original text to anchor interpretations so that we don't get involved in disagreements that arise out of translation discrepancies. 
4. This is a reference to the first article of the Mechanism series. Would probably be good to add a hyperlink whenever I refer to it.
5. That should be clear from what I write in the first article. I explain that the immediate identity is about the Concept. Let me know if you think that's unclear.

Reply:
1. I think will be sorted by the folder structure you set up. ✅
2. Well, if you mean to target the category in a technical sense, we can actually make use of the `code` formatting. It's what programmers use to highlight pieces of code. Since our texts will have no code to highlight, but we will highlight something technical, I thought it could be cool to appropriate it for ourselves. `code` formatting puts a text in courier font with a shaded background. I've used it myself in the writing I've done and it makes sense and looks good. You can make `code formatted text` by enclosing any bit with backticks " ` " <-- that thing in the middle there. 
Another cool thing is formatted code blocks. I've been using these to highlight a quotation with technical intent. You start a line with three backticks and then end the bit with another three with a new line at the end.
```md
In its indeterminate immediacy it is equal only to itself
and also not unequal with respect to another;
it has no difference within it, nor any outwardly.
```
What do you think?

3. We need to agree on a citation system. The encyclopaedia should adhere to one standard. I think MLA looks and works fine with inlined references. I think we should avoid using footnotes as much as possible, since they will be very hard to maintain (we don't enjoy an automatic footnote system here right now). 

4. Yes, either a link or briefly include the point. 
5. Ok ✅

Reply:
2. That sounds like a very nice function. I'll look to implement it in the article. 
3. Yes, MLA is easy-peasy. We can avoid footnotes, but we will need some acceptable way of making notes within the article. Maybe we created indented paragraphs in bold?
4. I think a link would be sleeker.

Reply:
3. There is footnote functionality, it's just that it should be used sparingly because we will need to manually update potentially all footnotes whenever there's a change. 
But I also wonder if one needs to go on lengthy tirade about something in a footnote, then either that should be important enough for the main body or be separated out into a new article or discussion (which one could link to). This is something we can embrace instead.

Reply:
OK. I've done the MLA referencing, and I've done the three ticks addition, too!